src/overlays/actors/ovl_En_Dnb/z_en_dnb.c

```diff
@@ -9,7 +15,24 @@ void EnDnb_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnDnb_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnDnb_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-#if 0
+s32 func_80A5086C(EnDnbStruct* arg0);
+s32 func_80A507C0(EnDnbStruct* arg0, Vec3f arg1, Vec3f arg2, u8 arg3, f32 arg4, f32 arg5);
+s32 func_80A50950(EnDnbStruct* arg0, GlobalContext* globalCtx);
+
+extern CollisionHeader D_06004D8C;
+extern Vec3s D_06004710[];
+extern Gfx D_06004638[];
+extern u64 D_0408F7E0[];
+extern u64 D_0408F3E0[];
+extern u64 D_0408EFE0[];
+extern u64 D_0408EBE0[];
+extern u64 D_0408E7E0[];
+extern u64 D_0408E3E0[];
+extern u64 D_0408DFE0[];
+extern u64 D_0408DBE0[];
```

Comment:
Two things:
- It seems like 04 symbols are externed in variables.h, so I think they should be externed there instead of here?
- We don't type textures as u64 any more: we discovered some objects where this wouldn't match. For now we have the type `TexturePtr` until we decide what to do generally.